health/api/v1/health_.py

```diff
@@ -117,19 +134,50 @@ def get_health(region):
         "projects": {}
     }
 
-    def convert_(data, field):
-        result = []
-        for d in data["buckets"]:
-            result.append([d["key_as_string"], d[field]["value"]])
-        return result
-
     for project in r.json()["aggregations"]["projects"]["buckets"]:
         result["project_names"].append(project["key"])
         result["projects"][project["key"]] = {
             "fci": project["avg_fci"]["value"],
-            "fci_score_data": convert_(project["data"], "fci"),
-            "response_time_data": convert_(project["data"], "response_time"),
-            "response_size_data": convert_(project["data"], "response_size")
+            "fci_score_data": convert(project["data"], "fci"),
+            "response_time_data": convert(project["data"],
+                                          "response_time"),
+            "response_size_data": convert(project["data"],
+                                          "response_size")
+        }
+
+    return flask.jsonify(**result)
+
+
+@health.route("/health", defaults={"period": "day"})
+@health.route("/health/<period>")
+def get_overview(period):
+
+    period, interval = get_period_interval(period)
+    query = get_query(
+        period, interval, aggs_name="regions", aggs_term="region")
+
+    request = config.get_config()["backend"]["elastic"]
+    r = requests.get("%s/_search" % request, data=json.dumps(query))
+
+    if not r.ok:
+        logging.error("Got {} status when requesting {}. {}".format(
+            request, r.text))
+        raise RuntimeError(r.text)
```

Comment:
we should use flask.abort() here (it's specially designed to stop request in generic way), I will change it in next patch.